cetmix_tower_yaml/security/cx_tower_yaml_wizard_access_rules.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo noupdate="1">
+
+        <!-- cx.tower.yaml.export.wiz -->
+        <record id="rule_cx_tower_yaml_export_wiz_creator_only" model="ir.rule">
+            <field name="name">Creator only</field>
+            <field name="model_id" ref="model_cx_tower_yaml_export_wiz" />
+            <field name="global" eval="True" />
+            <field name="domain_force">[('create_uid', '=', user.id)]</field>
+            <field name="perm_read" eval="True" />
+            <field name="perm_create" eval="True" />
+            <field name="perm_write" eval="False" />
```

Comment:
Why is this false? How you suppose the person who opened a wizard will put values into it? Same with all other wizards.